ui/src/components/FeedMewWrapper.vue

```diff
@@ -29,7 +29,7 @@
   <q-item-section>
     <div>
       <q-btn
-        @click="likeMew"
+        @click="toggleLickMew"
       >
         like ({{ mew.likes.length }})
```

Comment:
I would suggest changing these instances (and all others) of like to lick so we have consistency